tests/components/motioneye/test_camera.py

```diff
@@ -114,7 +114,7 @@ async def test_setup_camera_new_data_same(hass: HomeAssistant) -> None:
     """Test a data refresh with the same data."""
     client = create_mock_motioneye_client()
     await setup_mock_motioneye_config_entry(hass, client=client)
-    async_fire_time_changed(hass, dt_util.utcnow() + DEFAULT_SCAN_INTERVAL)
+    async_fire_time_changed(hass, dt_util.utcnow() + DEFAULT_SCAN_INTERVAL)  # type: ignore[no-untyped-call]
```

Comment:
Maybe just add a type hint to `async_fire_time_changed`?

Reply:
That is indeed a much better idea :-)